python/dgl/data/tu.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import
+import numpy as np
+import dgl
+import os
+
+from dgl.data.utils import download, extract_archive, get_download_dir
+
+
+class TUDataset():
+    """
+    TUDataset contains lots of graph kernel datasets for graph classification.
+    Use provided node feature by default. If no feature provided, use one-hot node label instead.
+    If neither labels provided, use constant for node feature.
+
+    :param name: Dataset Name, such as `ENZYMES`, `DD`, `COLLAB`
+    :param use_pandas: Default: False.
+        Numpy's file read function has performance issue when file is large,
+        using pandas can be faster.
+    :param hidden_size: Default 10. Some dataset doesn't contain features.
+        Use constant node features initialization instead, with hidden size as `hidden_size`.
```

Comment:
Briefly describe how node features will be stored in the graph (if edge features are available, please do that as well). Also describe what each sample looks like (e.g. a tuple of graph and label).

Reply:
Added more description at `__getitem__` method.